src/index.ts

```diff
@@ -209,45 +209,8 @@ export class Squid {
 
   public async getRoute(params: GetRoute): Promise<RouteResponse> {
     this.validateInit();
-
     const response = await this.axiosInstance.get("/api/route", { params });
```

Comment:
I guess the return type is "any" here, maybe good to force type with the response type ?
What happens if it fails, should we try catch here, or the getRoute calls are try catched ?